tests/virt_autotest/xen_guest_irqbalance.pm

```diff
@@ -156,15 +160,30 @@ sub prepare_guest_for_irqbalance {
 
 }
 
+# ping multiple URL to get enough vif interrupts to distribute on multiple IRQs
+sub generate_vif_interrupts_in_guest {
+    my $vm_name = shift;
+
+    #interrupts are handled by different vif IRQs from different sources
+    #and always the same IRQ handle the interrupts from same sources
+    my @ping_url_list = qw(download.opensuse.org 8.8.8.8 xen.org opensuse.org libvirt.org suse.com www.perl.org);
+    my $gateway = script_output("ssh root\@$vm_name \"ip r | grep 'default via' | cut -d ' ' -f3\"");
+    push @ping_url_list, $gateway;
+    foreach my $url (@ping_url_list) {
+        script_run("ssh root\@$vm_name \"ping -f -c 20 $url\"");
```

Comment:
Suggest to let the ping run in background with a "&" at the end of the command, so that mulitile ping can go parallelly and save test time.

Reply:
it does not take too much time for each ping, only 0.1s(https://openqa.nue.suse.com/tests/8279264#step/xen_guest_irqbalance/323). if it runs in backgroud, the result will not be shown in screenshots, sometimes the screenshots are helpful for test debugging.